src/main/java/impl/weather/YearTemperatureStatsImpl.java

```diff
@@ -0,0 +1,59 @@
+package impl.weather;
+
+import api.weather.DayTemperatureInfo;
+import api.weather.YearTemperatureStats;
+
+import java.time.Month;
+import java.util.*;
+import java.util.List;
+
+public class YearTemperatureStatsImpl implements YearTemperatureStats {
+    private final Map<Month, Map<Integer, DayTemperatureInfo>> status = new HashMap<>();
+    private final Average average = new Average();
```

Comment:
I meant that it would be better to create one class that contains information about the month, rather than three classes, each of which contains a map